src/commands/misc/helpRemember.js

```diff
@@ -0,0 +1,37 @@
+const { EmbedBuilder } = require('@discordjs/builders')
+
+module.exports = {
+    name: 'help-remember',
+    description: 'List of remember commands',
+
+    callback: (client, interation) => {
+        
+        const embed = new EmbedBuilder()
+        .setTitle('Remember Commands')
+        /*.setDescription('List of server commands')*/
+        .addFields(
```

Comment:
Is it possible to add fields dynamically so the name is the name of the command and the value is the description? I feel it would be easier as we add or remove commands from the bot

Reply:
Agreed that we should figure a way to do this dynamically, or at least by keeping the name/description with each command and pulling it in here (so we're not duplicating that data in the repo).

Reply:
For now I suggest leaving it as it is.